build/conformance.textproto

```diff
@@ -329,6 +329,19 @@ requirement: {
   whitelist_regexp: "test/test/util/util.js"
 }
 
+# Disallow the general use of AbortController, which is not available on all
+# supported platforms.
```

Comment:
Perhaps this should mention that the exception in the fetch plugin exists because that class is disabled if AbortController isn't defined?

Reply:
I added a bit of explanation.